### DISCUSSION TITLE
指定日期面板的时间选择范围

### DISCUSSION THREAD

**Discussion by guaijie** [ant-design/ant-design#46496] [Q&A]
![image](https://github.com/ant-design/ant-design/assets/30885718/87ccbe6d-6f9f-4f15-b55a-1fff0c62b45f)
如上图所示，我通过样式来隐藏年份的显示和对年份的选择，但我无法通过设置事件和属性（没有这些属性和事件）将面板的月份限制在 1 - 12 月份。因为我要求日期选择器不关心年份，它用于选择每年的具体的某个时间点。因此要求该日期面板必须显示闰年的 1- 12 月份。问题是现在我总能通过上面的月份选择器不断跳转直至下一年。而且无法指定日期选择面板是闰年

**Comment by zombieJ** [comment]
重构中，之后可以受控 `pickerValue` 把这个面板范围限死。

**Reply by guaijie** [reply]
panelRender 这个属性现阶段有办法实现上述功能吗

**Comment by guaijie** [comment]
> 重构中，之后可以受控 `pickerValue` 把这个面板范围限死。
4.x 也会加上 pickerValue 这个属性吧
